security/pkg/registry/kube/service.go

```diff
@@ -76,13 +75,26 @@ func (c *ServiceController) Run(stopCh chan struct{}) {
 }
 
 func (c *ServiceController) serviceAdded(obj interface{}) {
-	c.addFunc(obj.(*v1.Service))
+	svc := obj.(*v1.Service)
```

Comment:
You might also want to add the RBAC rules for K8s services at somewhere here https://github.com/istio/istio/blob/1f601f11ad63e27c730620b83a1b1506c5452c08/install/kubernetes/templates/istio-rbac-beta.yaml.tmpl#L69

Reply:
Will do. thanks for reminding